### PR TITLE
tool for creating worker nodes

### PR DESCRIPTION
**Pull request by dhellmann**

### CODE REVIEW

**Comment by russellb** [comment]
Hm, this seems to be OpenShift specific.  Should it go somewhere else, instead?

**Comment by dhellmann** [comment]
I removed the bits that are openshift-specific. We want folks to use the actuator anyway.

File: docs/dev-setup.md [hunk 1]
```diff
@@ -111,5 +111,34 @@ spec:
...
+
+---
+apiVersion: metalkube.org/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: worker-99
+spec:
+  online: true
+  bmc:
+    address: 1.2.3.4
```
Comment (russellb) [inline docs/dev-setup.md]: I guess it assumes IPMI without a prefix?

**Comment by russellb** [comment]
lgtm

File: docs/dev-setup.md (hunk 1 above)
Comment (dhellmann) [reply docs/dev-setup.md]: Yes, see https://github.com/metalkube/baremetal-operator/blob/master/pkg/bmc/access.go#L49

**Pull request merged**